### ISSUE TITLE
Fix logging

### ISSUE THREAD

**Issue by kmcquade** [salesforce/cloudsplaining#130]
Right now, we use [click_log](https://github.com/click-contrib/click-log) to set the log formatter. However, I noticed that the logger only shows up when it is in the command file.

## Steps to reproduce

```bash
 cloudsplaining scan-policy-file --input-file examples/policies/wildcards.json -v debug
```

The only logging output that shows up is this:

```
debug: Opening examples/policies/wildcards.json
```

That is expected, since it comes from `cloudsplaining.command.scan_policy_file`. 

However, there are a lot of other debug messages that should show up. For example, the `scan_policy` command leverages the `cloudsplaining.scan.policy_document.PolicyDocument` class [here](https://github.com/salesforce/cloudsplaining/blob/master/cloudsplaining/command/scan_policy_file.py#L142) but if you look at the actual policy_document.py file, you will see that there are a lot of other lines with `logger.debug` but those logging messages do not show up. For example: 

https://github.com/salesforce/cloudsplaining/blob/759e0bbc552b5d12d836a79e43c2ee947a8772c7/cloudsplaining/scan/policy_document.py#L184-L189

It would be really helpful if we could have the logger show up consistently so users know what is going on if there are issues.

**Comment by reetasingh** [comment]
@kmcquade I think thats because the handler is configured like that https://github.com/click-contrib/click-log/blob/0d72a212ae7a45ab890d6e88a690679f8b946937/click_log/core.py#L53

**Comment by reetasingh** [comment]
@kmcquade I have made an attempt at fixing this https://github.com/salesforce/cloudsplaining/pull/135

**Comment by kmcquade** [comment]
Fixed by #135
